src/actions/segment/segment.ts

```diff
@@ -215,7 +215,7 @@ export class SegmentAction extends Hub.Action {
   }
 
   protected segmentClientFromRequest(request: Hub.ActionRequest) {
-    return new segment(request.params.segment_write_key)
+    return new segment(request.params.segment_write_key, {flushAt: 500})
```

Comment:
Can we put this in a separate PR for clarity?

Reply:
Or at least its own commit

Reply:
for sure I'll take that out for this

Reply:
or rebase the whole thing so that the node upgrade and the segment flush are separate things.

and i'd still like to make this an environment variable with a default 500

Reply:
ok cool yea a seperate PR is best that way this one can be squashed